docs/Demo.md

```diff
@@ -1,447 +1,409 @@
+    extraEnv:
+    - name: LOG_LEVEL
+      value: DEBUG
+    - name: NAMESPACES
+      value: mongo-persistent
+  - app_name: committime-exporter
+    exporter_type: committime
+    extraEnv:
+    - name: LOG_LEVEL
+      value: DEBUG
+    - name: NAMESPACES
+      value: mongo-persistent
+  - app_name: failure-exporter
+    exporter_type: failure
+    extraEnv:
+    - name: LOG_LEVEL
+      value: DEBUG
+    - name: PROVIDER
+      value: GitHub
+    - name: TOKEN
+      value: ghp_J<snip>
+    - name: PROJECTS
+      value: <your_org>/mig-demo-apps
+```
 
-* Now refresh the Grafana dashboard and you should see the Change Failure Rate go up.
+> Note: Documentation regarding values.yaml can be found in the [Configuration ](Configuration.md) section.
 
-![change_failure_rate_1](img/change_failure_rate_1.png)
+2. Apply the updated values for Pelorus.
+```
+helm upgrade pelorus charts/pelorus --namespace pelorus --values /var/tmp/values.yaml
+```
+> **Note:** Allow time for the failure exporter pod to build and deploy.
 
-* Let's now create a non critical bug. A bug that does not indicate a deployment failure in your todolist application.  A the bug label however do *not* add the application label
+1. Check the output from the failure exporter and that no bugs are found.
+```
+curl $(oc get route -n pelorus failure-exporter -o=template='http://{{.spec.host | printf "%s\n"}}')
+```
 
-![issue_untagged](img/issue_2_non_deployment.png)
+### GitHub Issues
+Pelorus uses two tags to determine if a GitHub issue is associated with the todolist-mongo application and includes the default `bug` tag. Pelorus requires that all issues associated with a particular application be labeled with the app.kubernetes.io/name= label by default. This works the same way as the deployment configuration.
 
-* Ensure that issue #2 is not impacting our failure rate metric by curl-ing the output of the failure exporter.
+**Procedure**
+1. Navigate to https://GitHub.com/your_org/mig-demo-apps/issues.
+1. Set the required GitHub issue tags:
+    * `bug`
+    * `app.kubernetes.io/name=todolist`
 
-  **command:**
+![GitHub_start](img/GitHub_issues_setup.png)
 
-    curl $(oc get route -n pelorus failure-exporter -o=template='http://{{.spec.host | printf "%s\n"}}')
 
-*  Issue #1 should be found in the output of the curl.  Issue #2 will not be registered as a deployment failure because the issue is *not* tagged with `app.kubernetes.io/name=todolist`
+## Creating GitHub issues
+Follow the steps below to create an issue in GitHub and set the appropriate labels. Pelorus will register an issue as a deployment failure only if it is labeled as a `bug` and labeled with the application name `app.kubernetes.io/name=todolist`.
 
-Notice the message `failure_creation_timestamp`.  This indicates the time the issue was created.
+**Procedure**
+1. Create a GitHub issue and label it appropriately to register a failure.
 
-  **output:**
+![GitHub_issue1](img/GitHub_issue_1.png)
```

Comment:
this image did not render

Reply:
Changed github to lower case which should fix it. -- I made a total rookie mistake with find and replace.  I was working late but I should know better.